### ISSUE TITLE
Unnatural spacing in Exported PDF

### ISSUE THREAD

**Issue by AnselGitAccount** [shd101wyy/markdown-preview-enhanced#263]
**The spacing in Exported PDF is unnatural.** 
Steps to export a PDF:
1. Right click on preview;
2. Pick Export to Disk;
3. Choose PDF;

**For example, in preview window:**
![image](https://cloud.githubusercontent.com/assets/16124266/22747502/3fc966c4-edf5-11e6-802f-17ec9b521cbb.png)

**And in exported PDF:**
![image](https://cloud.githubusercontent.com/assets/16124266/22747559/60f15546-edf5-11e6-9749-e5ae792a8b07.png)

**Comment by AnselGitAccount** [comment]
I am using Fedora 24 btw.

**Comment by shd101wyy** [comment]
Hi @AnselGitAccount 

Pardon me, I can't see the difference clearly 😂 . Maybe the font size is different?

Thanks

**Comment by AnselGitAccount** [comment]
The spacing between characters is uneven.
For example, compare the word "Enhanced". The spacing between 'E' and 'n' is not the same as between 'n' and 'h' in the exported PDF. The text looks very "weird" .

**Comment by shd101wyy** [comment]
Ok I see.

**Comment by AnselGitAccount** [comment]
Thank you. Please let me know the cause. I am curious myself.

**Comment by shd101wyy** [comment]
I just tried on my Mac, and the pdf file looks fine.  
![screen shot 2017-02-08 at 4 30 28 pm](https://cloud.githubusercontent.com/assets/1908863/22760409/f422a510-ee1b-11e6-9b4e-f8ad3890bece.png)  

So I am wondering if this issue is caused by your pdf viewer?  
I will also attach the pdf file here.  
[test.pdf](https://github.com/shd101wyy/markdown-preview-enhanced/files/762121/test.pdf)

**Comment by AnselGitAccount** [comment]
Your test.pdf looks fine to me on my fedora 24. 
Here is what i generated.
[sdag.pdf](https://github.com/shd101wyy/markdown-preview-enhanced/files/762129/sdag.pdf)

**Comment by shd101wyy** [comment]
Can you try `Open in browser`. Then print as pdf from browser to see if the same issue happens?

**Comment by AnselGitAccount** [comment]
The text in browser look fine. But the pdf exported from browser has the same issue.

**Comment by shd101wyy** [comment]
This package calls browser api to print pdf. 
Therefore this issue might be caused by the browser.  
Could you please try print as pdf any other webpages (not from `Open in browser`, just any random webpage like www.google.com).  
If this issue still happens, then it is browser issue.  Otherwise it is the package issue.

**Comment by AnselGitAccount** [comment]
Same a random webpage to PDF has the same issue.
How do i fix the browser issue?  I am using Chrome as default.

**Comment by shd101wyy** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=72017  
Try change your system front? I am not sure if that would help or not :(

**Comment by AnselGitAccount** [comment]
I tried both sans and serif on Chrome. Neither works. 
Only set default browser to Firefox helps.

**Comment by shd101wyy** [comment]
Sorry I am not able to help in this case. As atom is based on `chromium`, that means the print of chromium doesn't work properly on fedora :(  
So the only way to get around is to `Open in browser` (not chrome) and then print as pdf.

**Comment by AnselGitAccount** [comment]
Yes, in my case (Fedora), the only way to get around is to Open in browser (Firefox) and then print as pdf.

**Comment by shd101wyy** [comment]
or you can try `phantomjs` instead

**Comment by AnselGitAccount** [comment]
**phantomjs** does not work either.

**Comment by shd101wyy** [comment]
:(
